src/be/app.py

```diff
@@ -28,16 +32,50 @@ def static(path: str):
         path=_fe_dir / path,
     )
 
-
 @app.get("/api/v1/status")
 def status():
+    #Use the `bulb` variable to get the properties of the LED strip, such as its name, power status, brightness, and color. Return these properties as an instance of the `Status` class.
+    properties = bulb.get_properties()
+    bulb_name = properties['name'] or "Unknown"
+    power_status = properties['power']
+    brightness = properties['bright']
+    colour = _int_to_colour(properties['rgb'])
+    print(f"Bulb name: {bulb_name}, Power status: {power_status}, Brightness: {brightness}, Colour: {colour}")
     return Status(
-        bulb_name="TODO",
+        bulb_name = bulb_name,
+        power_status = power_status,
+        brightness = brightness,
+        colour = colour
     )
 
 @app.post("/api/v1/turn_on")
 def turn_on():
-    # TODO turn bulb on here
-    return "Hello from the server! implement this function"
+    bulb.turn_on() 
+    return bulb.turn_on()
+
+@app.post("/api/v1/turn_off")
+def turn_off():
+    bulb.turn_off()
+    return bulb.turn_off()
```

Comment:
Current you're calling the `bulb.turn_off()` function twice - this isn't need, and you could have this function just like:

```suggestion
def turn_off():
    return bulb.turn_off()
```

This should also be done to the `/api/v1/turn_on` endpoint function

Extra explanation:
The first time you're calling it, and ignoring the output (line 58), as you're not assigning the output to a variable. The second time (line 59) you're returning the output of `bulb.turn_off()` from the outer function. We only need to call it once to have the light actually turn off, so line 58 is not needed

Reply:
Done